bin/run_ingestion.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
```

Comment:
Suggest `set -euxo pipefail` for [reasons](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/)

Reply:
I see that we are inspecting status later on to send email. So this may not be useful...